nixos/modules/services/audio/roon-server.nix

```diff
@@ -33,9 +47,8 @@ in {
       serviceConfig = {
         ExecStart = "${pkgs.roon-server}/opt/start.sh";
         LimitNOFILE = 8192;
-        DynamicUser = true;
-        SupplementaryGroups = "audio";
-        StateDirectory = name;
```

Comment:
Just noticed this, why remove this?

Reply:
It was complaining on startup when I left the StateDirectory bit in there. Not sure why.

Reply:
Without this, there can't really be any place for it to store data, so this should definitely be looked into further

Reply:
I removed it initially because it threw an error with it when I tested before. After some more troubleshooting, I think it's because I left the directory structure the same as when it was dynamic user based. In this configuration, the actual directory lives under `/var/lib/private` and the path `/var/lib/roon-server` is a symlink to the `roon-server` subdirectory in private. Moving it into the actual place fixed the error on launch.

Reply:
Hm I see, that makes me worried that systemd doesn't handle `DynamicUser` -> normal user transitions at all..

I guess it's fine anyways for this service since it's so new and didn't reach stable yet